etc/scripts/docenizer.py

```diff
@@ -12,7 +15,7 @@
 
 parser = argparse.ArgumentParser(description='Docenizes HTML version of the official Intel Asm PDFs')
 parser.add_argument('-i', '--inputfolder', type=str,
-                    help='Folder where the input files reside as .html. Default is current folder', default='./')
+                    help='Folder where the input files reside as .html. Default is current folder')
```

Comment:
Maybe the default in this could be the value for which we download the zip by default? That way, thing will "just" work!

Reply:
I don't get what you mean here. Can you give an example of what the "value" would be?

Reply:
Sure!
If you add a new argument for the download folder like `parser.add_argument('-d', '--downloadfolder', type=str, help='...', defaults='asm-docs')` then use the same default for the input folder argument (`'asm-doc'`), changing the logic around a bit, so it goes for the download if and only if it could not find the files from  `--inputfolder` path. I hope it makes some sense, but I think I explained myself a bit too bad hah

Reply:
Oh I get it. That's a nice move, indeed :+1: